notify.go

```diff
@@ -18,7 +18,7 @@ type Notify struct {
 // ErrSendNotification signals that the notifier failed to send a notification.
 var ErrSendNotification = errors.New("send notification")
 
-// New returns a new instance of Notify. Defaulting to being not disabled
+// New returns a new instance of Notify. Defaulting to being not disabled.
 func New() *Notify {
 	notifier := &Notify{
 		Disabled:  defaultDisabled,
```

Comment:
Is this Disabled functionality ever used?

Reply:
Personally I use it to disable notifications during runtime of my servers. I will leave this in for now, might be removed in the future. Thanks for the review!